jupyterlab/jupyterlab_jupytext/__init__.py

```diff
@@ -1,24 +1,12 @@
-"""Read and write Jupyter notebooks as text files"""
+"""Jupyter server and lab extension entry points"""
 
-from .formats import NOTEBOOK_EXTENSIONS, get_format_implementation, guess_format
-from .jupytext import read, reads, write, writes
-from .reraise import reraise
-from .version import __version__
+from jupytext.reraise import reraise
 
 try:
-    from .contentsmanager import build_jupytext_contents_manager_class
+    from jupytext.contentsmanager import build_jupytext_contents_manager_class
 except ImportError as err:
     build_jupytext_contents_manager = reraise(err)
 
-try:
-    from .contentsmanager import TextFileContentsManager
-except ImportError as err:
-    TextFileContentsManager = reraise(err)
-
```

Comment:
Oh sorry I've not been clear in my previous comment.

We need to keep the import `from .contentsmanager import TextFileContentsManager` because some people are still using `c.NotebookApp.contents_manager_class="jupytext.TextFileContentsManager"` in their config.

So maybe we revert this change and I open an issue to look into whether we still have use of `reraise` here?

Reply:
No problem, sure, I will do it.

Reply:
Actually looking at it again,  `jupytext` is providing [`TextFileContentsManager`](https://github.com/mwouts/jupytext/blob/b483de082c40c57faed507907298e924dbcc8d55/src/jupytext/__init__.py#L13-L16) so users will be still able to use `c.NotebookApp.contents_manager_class="jupytext.TextFileContentsManager"`. The current module you are referring to is `jupyterlab-jupytext`. Can you confirm it, please?

Reply:
Oh you''re right! I got confused by the GitHub interface, which compares the `__init__.py` for `jupytext` with the one for `jupyterlab-jupytext`. Thanks for double checking!